lib/ast-converter.js

```diff
@@ -268,6 +267,8 @@ function getTokenType(token) {
 
             case SyntaxKind.GetKeyword:
             case SyntaxKind.SetKeyword:
+            case SyntaxKind.TypeKeyword:
```

Comment:
They surfaced as bugs/missing checks when implementing the original work on this branch. I left them in because they seem like bugs waiting to happen, but I am happy to remove them if you feel that would be cleaner.